lib/runner/test-runners/default.js

```diff
@@ -107,6 +108,10 @@ class DefaultRunner {
       });
   }
 
+  isTestWorker() {
```

Comment:
this is already defined in [runner/cli/cli.js](https://github.com/nightwatchjs/nightwatch/blob/master/lib/runner/cli/cli.js#L66).

Reply:
I solved this by removing the definition in [runner/cli/cli.js](https://github.com/nightwatchjs/nightwatch/blob/master/lib/runner/cli/cli.js#L66) in favor of the one in the testRunner. Not sure if this is what you expected, but I couldn't find an easy way to remove the definition from the runner and use the one in cli.js.